ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-service-insertion.cfg

```diff
@@ -5,6 +5,20 @@ transceiver qsfp default-mode 4x10G
 hostname router-service-insertion
 !
 router service-insertion
+   connection connection1
+      interface Ethernet2/2.2 next-hop 10.10.10.10
+      monitor connectivity host host1
+   connection connection2
+      interface Tunnel2 primary
+      interface Tunnel3 secondary
```

Comment:
please configure at least one example without secondary and one without primary to verify this works.

Reply:
Added two new examples, one with only primary and one with only secondary.